### PR TITLE
fix client docs for 1.0.2 release

### PR DESCRIPTION
**Pull request by SachidanandAlle**

### CODE REVIEW

File: cpp-client/include/nvidia/aiaa/model.h [hunk 1]
```diff
@@ -63,6 +63,7 @@ struct AIAA_CLIENT_API Model {
   /// Image ROI size used while training [x,y,z,w] Format;  This shall be used for Annotation Models
   std::vector<int> roi;
 
+  /// Model Types support by AIAA
```
Comment (YuanTingHsieh) [inline cpp-client/include/nvidia/aiaa/model.h]: should this be supported?
but this is not important though

File: docs/tools.rst [hunk 2]
```diff
@@ -55,10 +58,43 @@ Example
...
-      -server http://10.110.45.66:5000/v1 \
+      -server http://0.0.0.0:5000 \
       -label spleen
 
 
+Session
+------------
+
+Provides implementation for ``nvidia::aiaa::Client::create_session()`` API.
+For more details refer `aiaa-session.cpp <https://github.com/NVIDIA/ai-assisted-annotation-client/blob/master/src/cpp-client/tools/aiaa/aiaa-session.cpp>`_
```
Comment (YuanTingHsieh) [inline docs/tools.rst]: links should be 
https://github.com/NVIDIA/ai-assisted-annotation-client/blob/master/cpp-client/tools/aiaa/aiaa-session.cpp
without "src"

File: docs/tools.rst [hunk 3]
```diff
@@ -55,10 +58,43 @@ Example
...
    nvidiaAIAAListModels \
-      -server http://10.110.45.66:5000/v1 \
+      -server http://0.0.0.0:5000 \
       -label spleen
 
 
+Session
+------------
+
+Provides implementation for ``nvidia::aiaa::Client::create_session()`` API.
```
Comment (YuanTingHsieh) [inline docs/tools.rst]: function is called "createSession" in cpp,
but this tool also provide implementation of "getSession" function
what do we want

File: cpp-client/include/nvidia/aiaa/model.h (hunk 1 above)
Comment (SachidanandAlle) [reply cpp-client/include/nvidia/aiaa/model.h]: fixed

File: docs/tools.rst (hunk 2 above)
Comment (SachidanandAlle) [reply docs/tools.rst]: fixed

File: docs/tools.rst (hunk 3 above)
Comment (SachidanandAlle) [reply docs/tools.rst]: fixed

**Pull request merged**